content/posts/2025/02/06/redesign/index.md

```diff
@@ -0,0 +1,76 @@
+  ![Series Toggle](Pile_Up_Series_toggle.png)
+
+## The New Calendar
+
+![Calendar](Calendar.png)
+
+We never planned for the calendar to play double duty as an index of games for a given day, but we found that many players used it this way. We've decided to make it serve this purpose more effectively by adding more structure and clarity to the list of games. We plan to make the index available for non-subscribers.
+
+## The New Clubs & Friends Page
+
+![Clubs and Friends](Clubs_and_Friends.png)
+
+Social presence on Puzzmo [changed a lot](https://blog.puzzmo.com/posts/2024/12/06/rm-chat/) over the past year. We removed chat, and we've been investing more in ambient and asynchronous social rather than real-time collaboration. Turns out, our players are a lot more interested in leaderboards, daily news, and profile stats than with than trying to collab on a typeshift. In the short term, as you can see fromm the mock-up above, we're making straightforward changes to simplify managing your friends and groups. Stay tuned for more experiments related to Puzzmo's social dimension later in the year.
+
+## Mobile Navigation
+
+![Mobile](Mobile.png)
+
+The mobile experience will be _mostly_ unchanged outside of a visual refresh. We do plan to remove the social tab, and drop the secondary toolbar from all screens.
+
+## The New Games Page
+
+I decided to save the best for last. We're redesigning the games page to bring more focus on the game itself, and to give players more control over which meta layers they want to experience alongside the game. This design maximizes the amount of space on the page that the game occupies, so you can play comfortably on any screen size.
+
+Everything on the game page is now unified under a single sidebar:
+
+![Bongo with Buzzmo](Bongo_with_Buzzmo.png)
+
+If you want to close the Buzzmo section, just press the section title:
+
+![Bongo closed tabs](Bongo_with_closed_tabs.png)
+
+If you want to minimize the sidebar completely, you can (if you are a subscriber—free accounts will still see an ad in the sidebar):
+
+![Bongo with collapsed sidebar](Bongo_collapsed.png)
+
+Finally, here's a preview of the new game complete sidebar - this is mostly the same as it is today:
+
+![](Pile_Up_Poker_Complete_Screen.png)
+
+## A Brief Technical Afterword
+
+This will be brief, for the frontend engineers in the audience.
+
+The original puzzmo.com website was written in React Native because we planned to release a cross-platform native app. We have since changed our mobile app strategy (we're excited about the new plans!), which means we don't need to use React Native Web. This is great. React Native for Web is an impressive project, but ultimately quite a leaky abstraction that we all found quite challenging to work with. We are switching to standard React (bundled with Vite), and this project is the first step toward making it happen.
```

Comment:
```suggestion
The original puzzmo.com website was [written in React Native](https://www.youtube.com/watch?v=2NItowAgfNA) because we planned to release a cross-platform native app. We have since changed our mobile app strategy (we're excited about the new plans!), which means we don't need to use React Native Web. This is great. React Native for Web is an impressive project, but ultimately quite a leaky abstraction that we all found quite challenging to work with. We are switching to standard React (bundled with Vite), and this project is the first step toward making it happen.
```